traits/has_traits.py

```diff
@@ -1989,6 +1988,8 @@ def configure_traits(
             **OK**), the new values are written to the file. If this parameter
             is not specified, the values are loaded from the in-memory object,
             and are not persisted when the dialog box is closed.
+            .. deprecated:: 6.0.0
```

Comment:
Please add a blank line above this one (and similarly below). Sphinx is fairly fussy about blank lines, and the deprecation note won't get rendered properly without an extra blank line here.

Reply:
Added in folllowing commit!